### PR TITLE
Remove IFrame from Prestashop

### PR DESCRIPTION
**Pull request by dairbuirabass**
## Description
Remove iframe from prestashop back office and use form controls to setup connection to Nosto.

## Checklist:
- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] I have assigned the correct milestone or created one if non-existent.
- [x] I have correctly labeled this pull request.
- [x] I have linked the corresponding issue in this description.
- [x] I have updated the corresponding Jira ticket.
- [x] I have requested a review from at least 2 reviewers
- [x] I have checked the base branch of this pull request
- [x] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
     "AFL-3.0"
   ],
   "require": {
-    "nosto/php-sdk": "^5.7"
+    "nosto/php-sdk": "dev-feature/remove-iframe-prestashop",
```
Comment (supercid) [inline composer.json:8]: Remember to update this one before merging it

File: controllers/admin/NostoIndexController.php [hunk 2]
```diff
@@ -56,25 +54,27 @@ public function getControllerUrls($employeeId)
...
     {
-        if ($account
-            && $account->isConnectedToNosto()
-            && Shop::getContext() === Shop::CONTEXT_SHOP
-        ) {
-            $currentUser = NostoCurrentUser::loadData();
-            $meta = NostoIframe::loadData();
-            return NostoSDKIframeHelper::getUrl($meta, $account, $currentUser);
+        if (Tools::getValue(NostoSDK::URL_PARAM_MESSAGE_CODE) === NostoSDK::CODE_ACCOUNT_CONNECT) {
+            if (Tools::getValue(NostoSDK::URL_PARAM_MESSAGE_TYPE) === NostoSDK::TYPE_SUCCESS) {
```
Comment (supercid) [inline controllers/admin/NostoIndexController.php:64]: You can combine those two if's to reduce nesting

**Review by supercid — APPROVED** [review]
Looks good. Can you deploy to plugintest so we can test?

**Review by supercid — CHANGES_REQUESTED** [review]
We need to have a look at this error though: 
image

File: composer.json (hunk 1 above)
Comment (dairbuirabass) [reply composer.json]: Updated. Merging the PR

**Comment by dairbuirabass** [comment]
> We need to have a look at this error though: image

This should be resolved now.

**Review by supercid — APPROVED** [review]

**Pull request merged**